starport_template/lib/stores/wallets_store.dart

```diff
@@ -50,4 +65,68 @@ class WalletsStore {
     wallets.value.add(creds.publicInfo);
     return creds.publicInfo;
   }
+
+  Future getCosmosBalances(BaseEnv baseEnv, String walletAddress) async {
```

Comment:
its a bit too much for a store to do this whole request building, lets extract it somewhere to a separate class, like:

```dart
CosmosBalances(baseEnv).getBalances();
```

Reply:
✅